### DISCUSSION TITLE
Floccus Android v5.0.6 hangs after installation

### DISCUSSION THREAD

**Discussion by Squidoid** [floccusaddon/floccus#1506] [General]
I've tried with F-Droid, Google Play and direct Github download versions and all of them hang whenever I start Floccus after install.

**Comment by PE5TT** [comment]
Me too, reverted to 5.0.5 witch works fine, I get blank screen or sometimes the floccus logo and it hangs. i am running LineageOS 20-20231014-microG-panther.

**Comment by marcelklehr** [comment]
v5.0.7 fixes this
